crates/fuel-core/src/executor.rs

```diff
@@ -3285,6 +3285,78 @@ mod tests {
             Ok(())
         }
 
+        fn on_chain_db_containing_message(
+            message: &Message,
+            da_block_height: u64,
+        ) -> Database<OnChain> {
+            let mut db = add_consensus_parameters(
+                Database::default(),
+                &ConsensusParameters::default(),
+            );
+            db.storage_as_mut::<Messages>()
+                .insert(message.id(), message)
```

Comment:
```suggestion
                .insert(message.nonce(), message)
```

May I ask you to rename this method please?=D `message.id()` for me means `MessageId` return type, while it is not true.

Reply:
There is already a `nonce` method that returns the same thing as `id`. Very silly.